src/Library/demos/Status Page/main.css

```diff
@@ -1,6 +1,6 @@
 box {
```

Comment:
By applying this class to the `box` name, it's cascading into all the child boxes and doesn't follow the GNOME HIG guidelines.

A better choice if you need a box-shadow is to use the [`card`](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/1-latest/style-classes.html#boxed-lists-cards) CSS class, which makes a nice rounded rectangle with a drop shadow.